asmcomp/x86_gas.ml

```diff
@@ -257,7 +258,7 @@ let print_line b = function
   | Comment s -> bprintf b "\t\t\t\t/* %s */" s
   | Global s -> bprintf b "\t.globl\t%s" s;
   | Long n -> bprintf b "\t.long\t%a" cst n
-  | NewLabel (s, _) -> bprintf b "%s:" s
+  | NewLabel (s, _) -> bprintf b "%s:\n" s
```

Comment:
What is the rationale for this added `\n`?

Reply:
Hmm. I think that's an artifact from when I tried inserting endbr64 here. I'll revert.